### PR TITLE
chore: update raster scheme, export details and remove WMTS_Layer from developer portal (MAPCO-2657)

### PR DESCRIPTION
**Pull request by asafMasa**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

Remove WMTS_Layer from developer portal and fix raster example
Update Export Details
Add transparency to raster catalog scheme

### CODE REVIEW

File: docs/assets/examples/raster/script.js [hunk 1]
```diff
@@ -107,6 +107,7 @@ const constructAndApplyLayer = (e) => {
     const footprint = xmlDoc.children[0].children[1].getElementsByTagName('mc:footprint')[0].textContent;
     const rectangle = generateRectangle(footprint);
 
+    // TODO: need to be fixed to capabilities url
```
Comment (syncush) [inline docs/assets/examples/raster/script.js:110]: we cant have todos in our portal

**Review by syncush — CHANGES_REQUESTED** [review]
also, why did the package-lock.json change? did you run `npm install` or `npm ci` ?

**Review by alebinson — APPROVED** [review]
looks good

File: docs/assets/examples/raster/script.js (hunk 1 above)
Comment (asafMasa) [reply docs/assets/examples/raster/script.js]: removed

File: docs/getting-started/raster/services/export_api.md [hunk 2]
```diff
@@ -4,7 +4,49 @@
  > :no_entry: **Authentication must be integrated in order to communicate with export service.**<br/>
 > **See the principles [here](/getting-started/raster/raster_authentication.md)**
 
-Export OpenAPI (Swagger)
+Export Service consists of /create POST message that allows users that have a valid token to create & download a GPKG file
```
Comment (syncush) [inline docs/getting-started/raster/services/export_api.md:7]: Why do we describe the open API that way? isn't it better to have a link for it?

**Review by syncush — CHANGES_REQUESTED** [review]
I am asking again, why did you update the packages? did you run npm install instead of `npm ci`?

File: docs/getting-started/raster/services/export_api.md (hunk 2 above)
Comment (asafMasa) [reply docs/getting-started/raster/services/export_api.md]: fixed

File: docs/getting-started/raster/services/export_api.md [hunk 3]
```diff
@@ -1,40 +1,46 @@
...
+
+
+### Important Notes
+1. The data is returned via callback response or on request response (structure in appendix 1)
+2. To download the data one needs to use authentication token
+3. To download the "metadta.json" file, write the GPKG url and change the sufix to ".json"
+
+
+### Appendix 1 - Callback response structure
     - requestId: uuid - unique identifier for export request - the field that was returned from orginal create request
```
Comment (syncush) [inline docs/getting-started/raster/services/export_api.md:44]: for the callback, I guess you send a JSON body, I would expect that you place your JSON schema here

**Comment by asafMasa** [comment]
> I am asking again, why did you update the packages? did you run npm install instead of `npm ci`?

I wanted to fix some vulnerabilities via audit-fix

File: docs/getting-started/raster/services/export_api.md [hunk 4]
```diff
@@ -1,40 +1,46 @@
...
-    - json object for newly created or "In progress" export task
+
+
+### Important Notes
+1. The data is returned via callback response or on request response (structure in appendix 1)
+2. To download the data one needs to use authentication token
+3. To download the "metadta.json" file, write the GPKG url and change the sufix to ".json"
+
+
+### Appendix 1 - Callback response structure
```
Comment (syncush) [inline docs/getting-started/raster/services/export_api.md:43]: it is the not the callback response, it is the callback request, since you init the request

**Comment by syncush** [comment]
> > I am asking again, why did you update the packages? did you run npm install instead of `npm ci`?
> 
> I wanted to fix some vulnerabilities via audit-fix

it should be in a different pull request

File: docs/getting-started/raster/services/export_api.md (hunk 4 above)
Comment (asafMasa) [reply docs/getting-started/raster/services/export_api.md:43]: its terminology as it is a response to the job export request - but i'll update

File: docs/getting-started/raster/services/export_api.md (hunk 3 above)
Comment (asafMasa) [reply docs/getting-started/raster/services/export_api.md:44]: added

**Review by syncush — APPROVED** [review]

**Pull request merged**